doc/tutorial.rst

```diff
@@ -586,8 +573,9 @@ The `static/js/src/simplevideo_edit.js` JavaScript file should look like this: :
           maxwidth: $(element).find('input[name=maxwidth]').val(),
           maxheight: $(element).find('input[name=maxheight]').val()
         };
+        runtime.notify('save', {state: 'start'});
         $.post(handlerUrl, JSON.stringify(data)).done(function(response) {
-          window.location.reload(false);
+          runtime.notify('save', {state: 'end'});
```

Comment:
This brings up a question. Should we cover how to access the studio view in the SDK? The SDK runtime doesn't have a reference to `notify`, but it does have the ability to render the studio view. @antoviaque Thoughts?